### PR TITLE
Exploration Update

### PR DESCRIPTION
**Pull request by SpadesNeil**
## Exploration Changes
- Added a huge area outside of the station walls to be explored.
- Random mobs spawn throughout the area. (ToDo in future: Make it so mobs replenish if no players are present.)
- Beach.dmm is Gateway ready.
- blackmarketpackers.dmm is Gateway ready.
- listeningpost.dmm is Gateway ready.

## Mob Changes
- Syndiefox can now survive in space.
- Geese can now cooperate. God help you.
- Fixed space bees not being retaliate mobs
- Fixed obsolete pathing of dragon mobs.
- Added space frog variant of frog that can survive in space. Probably needs new icons.
- Added feral version of otie, which will probably not enjoy pettings. Also it can survive outside on the surface.
- Several fixes with otie, including making the pathing of their mob type make sense. Pls no memes or canihascheezburger speak in item paths.

## Lore Changes
- Changed NSS Adephagia to be NSB Adephagia.
- Changed planets from 6 to 5. Nobody cared about Virgo-6 anyway so as far as I care it never existed.
- Elaborated on the Zorren in the Virgo Prime blurb.
- Virgo 3b changed to Virgo-3B for consistency and professional looking appearance.
- Virgo-4 is better defined as an ice world so that's what it is now.

## Misc. Changes
- Added a couple of guns to random gun spawner.
- Added .44 speedloader to random ammo spawner.
- Fixed exploit with .44 speedloaders and autolathe.
- Hotfixed .44 magazine icon until next Polaris update overrides it.
- Removed redundant magazine type from SVD list.
- Tried to fix stomach tiles not showing icons correctly in Dream Daemon.
- Deleted some dead code in station_vr.dm that was commented out anyway.

### CODE REVIEW

File: code/game/objects/random/random_vr.dm [hunk 1]
```diff
@@ -165,4 +170,18 @@
...
+	return	/obj/structure/trash_pile
+
+/obj/random/outside_mob
+	name = "Random Mob"
+	desc = "Eek!"
+	icon = 'icons/mob/screen1.dmi'
+	icon_state = "x"
+	spawn_nothing_percentage = 10
+
+/obj/random/cargopod/item_to_spawn()
```
Comment (SpadesNeil) [inline code/game/objects/random/random_vr.dm]: erm, this shouldn't be here

**Comment by SpadesNeil** [comment]
I'm expecting at least one Travis error.

**Comment by SilverTalismen** [comment]
With these changes to water fowl, Travis issues will be the *Geese* of our problems.

**Comment by SpadesNeil** [comment]
> With these changes to water fowl, Travis issues will be the Geese of our problems.

https://www.youtube.com/watch?v=0ufGsdlbwPI

**Comment by SpadesNeil** [comment]
Wait, it actually worked on the first try?

**Comment by SilverTalismen** [comment]
I'm sure nothing will go *Fowl* if this is merged, Might all be fine and not cause issues later on.

**Comment by SpadesNeil** [comment]
>I'm sure nothing will go Fowl if this is merged, Might all be fine and not cause issues later on.

https://www.youtube.com/watch?v=-NuONuafxwk

**Comment by SilverTalismen** [comment]
Well, Ace really wants me to get the *flock* out of here, so bye!~

**Comment by SpadesNeil** [comment]
reee

**Comment by SpadesNeil** [comment]
I like how it says when I delete other people's comments but not my own.

**Comment by SilverTalismen** [comment]
Aw all my jokes got deleted :C

**Pull request merged**